### PR TITLE
Update iterative training and pixel workflow

### PR DESCRIPTION
**Pull request by softcat477**
Update iterative training and pixel workflow

### CODE REVIEW

File: tutorial/iterative-training.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ parent: OMR Tutorial
 nav_order: 5
 ---
 # Iterative Training with Paco Classifier
-![](https://i.imgur.com/2onLoUw.png)
+<figure markdown="1">
+![]({{site.baseurl}}/assets/workflow-iterative-training.png)
+</figure>
 
 This page demosntrates steps to perform iterative training using `background_removal`, `Pixel_js`, `Training model of Patchwise Analysis of Music Document, Training`, and `Training model of Patchwise Analysis of Music Document, Classifying`. Overall, the iterative training process contains the following steps:
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:12]: "demonstrates" is misspelled

File: tutorial/iterative-training.md [hunk 2]
```diff
@@ -19,7 +21,9 @@ This page demosntrates steps to perform iterative training using `background_rem
 This step uses the [`background_removal`](../overview/document-analysis.md#background-removal) job to remove background of a manuscript. It provides two methods to do background removal: using `Sauvola's Threshold` or `SAE binarization.` In this example, we use `Sauvola's Threshold` and its default settings.
 
 The reuslt looks like this:
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:23]: The **result** looks like this:

File: tutorial/iterative-training.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ parent: OMR Tutorial
 nav_order: 5
 ---
 # Iterative Training with Paco Classifier
-![](https://i.imgur.com/2onLoUw.png)
+<figure markdown="1">
+![]({{site.baseurl}}/assets/workflow-iterative-training.png)
+</figure>
 
 This page demosntrates steps to perform iterative training using `background_removal`, `Pixel_js`, `Training model of Patchwise Analysis of Music Document, Training`, and `Training model of Patchwise Analysis of Music Document, Classifying`. Overall, the iterative training process contains the following steps:
 1. (*Iteration 1*) Annotate layers with `background removal` and `pixel`.
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:13]: For step 2: "go generate models with the **image** and layers you ..."

Step 3: "Send the generated models and new **image** to ..."
"The **classifier** predict**s** the layers **of this new image** for you.

Step 4: "until you get satisfying results (**i.e., when** the paco classifier...)"

File: tutorial/iterative-training.md [hunk 4]
```diff
@@ -19,7 +21,9 @@ This page demosntrates steps to perform iterative training using `background_rem
 This step uses the [`background_removal`](../overview/document-analysis.md#background-removal) job to remove background of a manuscript. It provides two methods to do background removal: using `Sauvola's Threshold` or `SAE binarization.` In this example, we use `Sauvola's Threshold` and its default settings.
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:21]: "to remove **the** background of **an image**.

File: tutorial/iterative-training.md [hunk 5]
```diff
@@ -94,4 +98,4 @@ The input for this job is a manuscript that we're going to classify and generate
 ## Iteration 2+
 For `2+` iterations, the workflow `Paco Trainer` and `Paco Classifier` is the same. The only difference is the `Background Removal` and `Pixel_js`.
 
-Basically, in `2+` iteration, we don't need the `Background Removal` job anymore. The predicted layers, output `Layer<i>` from `Paco Classifier` of the last iteration, are the input layers (`PNG-Layer<i> Input`) to `Pixel_js`. So we correct the predicted layers from the trained model of the previous, send all zip files we've used in the previous iteration to `Paco Trainer`, and add the new zip file that we've just corrected using `Pixel_js`.
\ No newline at end of file
+Basically, in `2+` iteration, we don't need the `Background Removal` job anymore. The predicted layers, output `Layer<i>` from `Paco Classifier` of the last iteration, are the input layers (`PNG-Layer<i> Input`) to `Pixel_js`. So we correct the predicted layers from the trained model of the previous, send all zip files we've used in the previous iteration to `Paco Trainer`, and add the new zip file that we've just corrected using `Pixel_js`.
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:101]: "So we correct the predicted layers from the trained model of the previous **iteration**, send all zip files we've used in the previous iteration to **the** `Paco TRainer` **job**, and ...

**Review by martha-thomae — CHANGES_REQUESTED** [review]
Thanks @softcat477! I will approve this PR after the few changes I indicated here.

File: tutorial/iterative-training.md [hunk 6]
```diff
@@ -5,21 +5,25 @@ parent: OMR Tutorial
...
+This page demonstrates steps to perform iterative training using `background_removal`, `Pixel_js`, `Training model of Patchwise Analysis of Music Document, Training`, and `Training model of Patchwise Analysis of Music Document, Classifying`. Overall, the iterative training process contains the following steps:
 1. (*Iteration 1*) Annotate layers with `background removal` and `pixel`.
-2. (*Iteration 1*) Use `Paco Trainer` to generate models with the manuscript and layers you just annotated.
-3. (*Iteration 1*) Send the generated models and new manuscripts to `Paco Classifier`. It predict layers for you.
-4. (*Iteration 2+*) Correct predicted layers. Repeat `step 1`~`step 3` until you get satisfying results (means the paco classifier is able to predict layers correctly).
+2. (*Iteration 1*) Use `Paco Trainer` to generate models with the image and layers you just annotated.
+3. (*Iteration 1*) Send the generated models and new image to `Paco Classifier`. The classifier predict layers of this new image for you.
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:15]: "predicts" with an "s" at the end

**Review by martha-thomae — CHANGES_REQUESTED** [review]
One super minor change that was missed and done.

File: tutorial/iterative-training.md [hunk 7]
```diff
@@ -5,21 +5,25 @@ parent: OMR Tutorial
...
+4. (*Iteration 2+*) Correct predicted layers. Repeat `step 1`~`step 3` until you get satisfying results (i.e., when the paco classifier is able to predict layers correctly).
 
 ---
 ## Iteration 1
 ### Background Removal.
-This step uses the [`background_removal`](../overview/document-analysis.md#background-removal) job to remove background of a manuscript. It provides two methods to do background removal: using `Sauvola's Threshold` or `SAE binarization.` In this example, we use `Sauvola's Threshold` and its default settings.
+This step uses the [`background_removal`](../overview/document-analysis.md#background-removal) job to remove the background of an image. It provides two methods to do background removal: using `Sauvola's Threshold` or `SAE binarization.` In this example, we use `Sauvola's Threshold` and its default settings.
 
-The reuslt looks like this:
```
Comment (martha-thomae) [inline tutorial/iterative-training.md:21]: Oh, and this one was missed too: "result"

**Review by martha-thomae — CHANGES_REQUESTED** [review]

File: tutorial/iterative-training.md (hunk 6 above)
Comment (softcat477) [reply tutorial/iterative-training.md]: Fixed in [0ced499](https://github.com/DDMAL/e2e-omr-documentation/pull/15/commits/0ced49911cd42bc3132f08ee2ae6ddb9c769019f)

File: tutorial/iterative-training.md (hunk 7 above)
Comment (softcat477) [reply tutorial/iterative-training.md:21]: This one has been fixed. The red one means this that lines has been deleted.

Comment (softcat477) [reply tutorial/iterative-training.md:21]: Fixed in here: https://github.com/DDMAL/e2e-omr-documentation/blame/0ced49911cd42bc3132f08ee2ae6ddb9c769019f/tutorial/iterative-training.md#L23

Comment (martha-thomae) [reply tutorial/iterative-training.md:21]: Oops, my bad! Didn't see the green line with the correction below.

**Review by martha-thomae — APPROVED** [review]

**Pull request merged**